app/models/user.rb

```diff
@@ -24,4 +24,15 @@ def favorited(post)
   def voted(post)
     votes.find_by(post_id: post.id)
   end
+
+  def self.top_rated
+  self.select("users.*") # Select all attributes of the user
+      .select("COUNT(DISTINCT comments.id) AS comments_count") # Count the comments made by the user
+      .select("COUNT(DISTINCT posts.id) AS posts_count") # Count the posts made by the user
+      .select("COUNT(DISTINCT comments.id) + COUNT(DISTINCT posts.id) AS rank") # Add the comment count to the post count and label the sum as "rank"
```

Comment:
Line is too long. [149/80]<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.